integration-tests/test_cdk_parallel.groovy

```diff
@@ -0,0 +1,70 @@
+
```

Comment:
Can we put this script somewhere outside of `integration-tests/`? Anywhere else is fine. Maybe `tests/` makes sense.

Everything in `integration-tests/` is runnable by any of us on our own laptops. That's true today and I'd like to keep it true. So Jenkins scripts gotta go elsewhere.

Reply:
Done with the move.

Also fixed the script to handle the lxdnode we have. Ideally we would have an lxd controller same as all the others so we would not need any extra care for lxd, but for now we need to send the job to the specific lxd slave.